dubbo-registry-extensions/dubbo-registry-nameservice/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.apache.dubbo.extensions</groupId>
+		<artifactId>dubbo-registry-extensions</artifactId>
+		<version>${revision}</version>
+	</parent>
+	<artifactId>dubbo-registry-nameservice</artifactId>
```

Comment:
rename `dubbo-registry-nameservice` to `dubbo-registry-rmq-nameservice` or `dubbo-registry-rocketmq-nameservice`